examples/next-js-cms-blog/lib/api.js

```diff
@@ -0,0 +1,39 @@
+import { Builder, builder } from '@builder.io/react';
+import { BUILDER_CONFIG } from './constants';
+
+builder.init(BUILDER_CONFIG.apiKey);
+Builder.isStatic = true;
+
+export function getAllPostsWithSlug() {
+  return builder.getAll(BUILDER_CONFIG.postsModel, {
```

Comment:
please try to follow the examples contribution docs here https://github.com/BuilderIO/builder/blob/main/examples/CONTRIBUTING.md
for example:
- let's remove all those utility functions and call the API directly in the page code
- add more comments with links to SDK reference and docs
- Remove all extraneous query params from api calls,